tests/bounds.c

```diff
@@ -228,7 +229,7 @@ void contiguous_on_buffer_boundary(void **state)
   zone_buffers_t buffers = { 1, &owner, &rdata };
 
   // generate zone file to parse
-  char *path = tempnam(NULL, "xbounds");
+  char *path = get_tempnam(NULL, "xbounds");
   assert_non_null(path);
   FILE *handle = fopen(path, "wb");
```

Comment:
The `path` is not unlinked, with like `remove_include(path);` or `unlink(path);` before `free(path);` at the end of the function.

Reply:
fixed